### PR TITLE
Get all fam and member endpoint/leslie

### PR DESCRIPTION
**Pull request by thereactgirl**
# PR Description
​
This PR includes new endpoints needed to return an entire household and is a part of the household percent complete feature that returns a percentage of how complete a guest intake form is.
​
​
## Type of Change
​
Types of Change:
- [x] New feature
- [] Refactoring / Bug fix
- [] Documentation update 
​
​
## Testing
​
- [x] Yes tested with Postman
- [] No
- [] Not needed
​
​
## Completeness
Is this feature complete
- [x] Feature Complete
- [] Feature Incomplete, user story/trello card linked
​
# PR Requirements
- [x] Clean and concise commenting
- [x] Clean and comprehensive documentation included
- [x] Conforms to engineering standards and checks
- [x] There are no merge conflicts
- [x] There are no warnings
- [x] Corresponding documentation elsewhere has been updated to reflect these changes
​
​

​
​
## Other
[Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)  
[Template Guidelines](https://www.notion.so/Pull-Request-Template-f9264f79e1b649b9845961b5aba3eaff)
​

### CODE REVIEW

**Comment by thereactgirl** [comment]
@Ehuntwork @abdimohamud @isaiah-j @KristineYW I know Slack is down but we still have a lot of work to do and this PR is ready for review. 😁

File: api/families/familiesRouter.js [hunk 1]
```diff
@@ -44,6 +46,32 @@ router.get('/:id/members', authRequired, function (req, res) {
...
+    .then((notes) => {
+      res.status(200).json(notes);
+    })
+    .catch((err) => {
+      res.status(500).json({ error: err.message });
+    });
+});
+
+// returns alll family info like vehicles as well as all guest info
+router.get('/:id/household', function (req, res) {
```
Comment (abdimohamud) [inline api/families/familiesRouter.js:63]: I think these endpoint needs to be authenticated before production, but I understand the functionality for this endpoint and like the comment as well the variable names you used for the function and parameters

File: api/families/familiesModel.js [hunk 2]
```diff
@@ -12,6 +12,10 @@ const findById = (id) => {
   return db('families').where({ id }).first().select('*');
 };
 
+const findFamilyByUserId = async (id) => {
+  return db('families').where({ user_id: id }).returning('*');
+};
+
```
Comment (abdimohamud) [inline api/families/familiesModel.js:18]: I'm think I'm kinda confused on how this function is async if the endpoint that uses this function isnt async as well, I've had some trouble in the past, but if it was able to work, fine by me! 👍

**Review by abdimohamud — APPROVED** [review]
Overall, looks good to me, and once those routes are authenticated we can push it to main

**Pull request merged**